### DISCUSSION TITLE
Next js Prerending issue

### DISCUSSION THREAD

**Discussion by jbsethi** [vercel/next.js#22760] [Help]
i have created to pages that are using `getServerSideProps` to fetch data and display it on the page. The problem is that when i visit the page the page doesn't load until the `getServerSideProps` returns. I have some hard code data on page like page title and some other things that i want to show regardless of api call. Is there a way to Render some part of page on user side before `getServerSideProps` is called.

![Selection_043](https://user-images.githubusercontent.com/27204988/109921939-6ee4bb80-7cde-11eb-9040-6a30d1223263.png)

**Comment by ErnestBrandi** [comment]
Hi, that does not make sense. Just fetch the data client side then (you could use SWR which is also made by Vercel).
